### PR TITLE
Construct a footer back to bugs and source for the app if possible.

### LINKED ISSUE #8: Provide links to the 'source' and 'bug tracker' for each app
It would be cool if 

1) fedmenu could guess which app it is being rendered on and then if we
2) include two new `bugs_url` and `source_url` fields in the `data.js` file, then we can
3) show a little footer in the menu saying "click here to get the source for this app or file a bug"

and we'll have a consistent place for people to look if they want to contribute to infra apps.

### PR DESCRIPTION
**Pull request by ralphbean**
This also includes the code from #18.

Fixes #8 for @nirik.

### CODE REVIEW

**Comment by ralphbean** [comment]
Sorry -- this got a little messy commit-wise.  The changes are all intentional, though.  They just should've been separated out into different PRs.

File: js/fedmenu.js [hunk 1]
```diff
@@ -108,10 +113,52 @@ var fedmenu = function(options) { $(document).ready(function() {
...
+            })
+        });
+        return found;
+    }
+
+    // Try to construct a little footer for the menus.
+    var add_footer_links = function() {
+        var site = get_current_site();
+        var content = "";
+        if (site != null && site.data.bugs_url != undefined && site.data.source_url != undefined) {
```
Comment (pypingou) [inline js/fedmenu.js:152]: Should we allow to specify only one of the two? (ie: specify a bugs_url but not the source_url or vice-versa)

**Comment by pypingou** [comment]
One question, otherwise :+1:

File: js/fedmenu.js (hunk 1 above)
Comment (ralphbean) [inline js/fedmenu.js:152]: Do we have a use case for that?

Comment (pypingou) [inline js/fedmenu.js:152]: For example pagure's footer only points to the sources, not to the issues

Comment (ralphbean) [inline js/fedmenu.js:152]: Hm, let's just show them both for now and we can make it more flexible if we want to later.

**Pull request merged**